style/learning-guide.less

```diff
@@ -3,8 +3,13 @@
   svg {
     width: 100%;
     .main-title{
-      font-size: 3px; 
+      font-size: 2.5px; 
```

Comment:
That was my bad.  I didn't notice that fonts should be rem.   (and you didn't catch it first during the earlier review  :stuck_out_tongue_winking_eye: )